### ISSUE TITLE
Problem during building

### ISSUE THREAD

**Issue by foXaCe** [oechslein/Hassio-Addons#1]
hello,
on my Rpi 4

 ERROR (SyncWorker_16) [hassio.docker.addon] Can't build 687586e7/armv7-addon-valetudo-mapper:0.2.0-1: The command '/bin/ash -o pipefail -c apk add --no-cache git=2.20.1-r0 npm=10.14.2-r0' returned a non-zero code: 2

**Comment by oechslein** [comment]
Are you using hassio or hassbian or something else?
I do have a Rpi4 with hassio and there it works. It first compiles the docker image on the rpi and that somehow fails on your machine.

**Comment by foXaCe** [comment]
Hassio

**Comment by impankratov** [comment]
Hello there,
Thank you for this mapper add-on.
I have same error with hassio on rpi 3b+:
```
20-01-28 20:02:50 INFO (SyncWorker_17) [hassio.docker.addon] Start build 687586e7/armv7-addon-valetudo-mapper:0.2.0-1
20-01-28 20:02:59 ERROR (SyncWorker_17) [hassio.docker.addon] Can't build 687586e7/armv7-addon-valetudo-mapper:0.2.0-1: The command '/bin/ash -o pipefail -c apk add --no-cache git=2.20.1-r0 npm=10.14.2-r0' returned a non-zero code: 1
```
@oechslein This part got my attention: `/bin/ash`. Shouldn't it be `/bin/bash` instead?

**Comment by oechslein** [comment]
I'm not too familiar with Docker, this line comes from 
`RUN apk add --no-cache git=2.20.1-r0 npm=10.14.2-r0`
in the Dockerfile.

I copied the config from here [https://github.com/Poeschl/Hassio-Addons/tree/master/ICantBelieveItsNotValetudo] and adjusted it to use rand256 version. I also added that the docker image is created and not downloaed from docker hub. Otherwise I would have to upload something to docker hub (like Poeschl does).

So until someone with more knowledge comes in, I cannot help you with it.
